app/views/pages/index.html.erb

```diff
@@ -1,4 +1,23 @@
 <div class="container text-center">
   <h1>Pages#index</h1>
   <p>Find me in app/views/pages/index.html.erb</p>
+  <% if current_user %>
+    <h2>Logged-in User:</h2>
+    <hr>
+    <%= current_user.username %><br>
+    <%= current_user.email %>
+    <h2>Welcome, stranger!</h2><br>
```

Comment:
If the current user is logged in we are saying welcome, stranger? This is also in your else block. Something doesn't add up.